components/Sidebar/ModuleList.tsx

```
@@ -191,23 +192,30 @@ export default function ModuleList({
+                      }}
+                      style={{ maxWidth: maxWidth }}
+                      className={cn(
+                        "w-full text-left px-2 py-2 text-sm rounded-md hover:bg-accent hover:text-accent-foreground flex items-center gap-2 group",
+                        checkIsActive(
+                          `/modules/${encodeModuleSlug(module.name)}`
+                        )
+                          ? "bg-accent text-accent-foreground font-regular border-r-4 border-primary shadow-sm"
+                          : "",
+                        collapsed && "justify-center"
+                      )}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)